docs/magus/skills/modifiers.md

```diff
@@ -8,19 +8,21 @@ Common modifiers can be unlocked by anyone in the skill tree.
 
 ### Focus
 
-Charges the technique by holding down the form button
+Charges the technique by holding down the form button.
 
 ### Multi
 
-Multiplies the technique by repeating the form button
+Multiplies the technique by repeating the form button.
 
 ### Directional
 
-Aims the technique in a certain direction by walking that way
+Aims the technique in a certain direction by walking that way.
+Additionally, the `shift`/`sneak` key makes it "lower", such as for
+a fire kick sweep, or block.
```

Comment:
Be consistent in how you format technique names: Ice Breath in the previous file, but fire kick here